src/core/agent/services/identifierService.ts

```diff
@@ -43,7 +43,7 @@ class IdentifierService extends AgentService {
     "Failed to rotate AID, operation not completing...";
   static readonly FAILED_TO_OBTAIN_KEY_MANAGER =
     "Failed to obtain key manager for given AID";
-  static readonly IDENTIFIER_NAME_TAKEN =
+  static readonly IDENTIFIER_NAME_TAKEN = // @TODO - foconnor: Can be removed after UI PR
```

Comment:
I didn't update the UI to stop expecting this error, but it's no longer used. Perhaps @sdisalvo-crd or @Sotatek-DukeVu can contribute these changes, and delete this line 46 here?

Reply:
@iFergal  Done.

Reply:
thank you!!